extensions/ql-vscode/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [UNRELEASED]
 
+- Removed "last updated" information and sorting from variant analysis results view. [#2637](https://github.com/github/vscode-codeql/pull/2637)
```

Comment:
Minor, it looks like the other entries below use "Remove", "Implement" etc instead of past tense
```suggestion
- Remove "last updated" information and sorting from variant analysis results view. [#2637](https://github.com/github/vscode-codeql/pull/2637)
```